client/components/foldable-card/docs/example.jsx

```diff
@@ -1,21 +1,22 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import React, { PureComponent } from 'react';
 
 /**
  * Internal dependencies
  */
 import FoldableCard from 'components/foldable-card';
 import Button from 'components/button';
 
-export default React.createClass( {
-	displayName: 'FoldableCard',
+export default class FoldableCardExample extends PureComponent {
+	static displayName = 'FoldableCardExample';
```

Comment:
Thanks for review!

`displayName` should be maintained in devdocs to keep readable names when production mangling is applied. [See README](https://github.com/Automattic/wp-calypso/tree/ef236e95b93170181de1169942d6f20b89dba35e/client/devdocs/design#component-name-convention).

I tested to make sure it's necessary for my own sake. It prevents this, so yep 😛 

![mangled](https://user-images.githubusercontent.com/841763/27828594-27ba128c-60bd-11e7-976c-c11e620ac60b.png)